tests/usb/usb.go

```diff
@@ -0,0 +1,141 @@
+package usb
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	expect "github.com/google/goexpect"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/types"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/api/usb/v1alpha1"
+	usbv1alpha1 "kubevirt.io/client-go/generated/kubevirt/clientset/versioned/typed/usb/v1alpha1"
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/tests"
+	"kubevirt.io/kubevirt/tests/console"
+	"kubevirt.io/kubevirt/tests/decorators"
+	"kubevirt.io/kubevirt/tests/framework/cleanup"
+	"kubevirt.io/kubevirt/tests/framework/kubevirt"
+	"kubevirt.io/kubevirt/tests/libvmi"
+	"kubevirt.io/kubevirt/tests/util"
+)
+
+var _ = Describe("[sig-compute-realtime][Serial]USB", Serial, decorators.SigCompute, func() {
```

Comment:
We should introduce `USB` label

Reply:
I took note to change this from `sig-compute-realtime`  to `sig-compute`. As adding label, should be similar to what Vasiliy did with `SEV`  in [this commit](https://github.com/kubevirt/kubevirt/commit/82e4c82971ce0d1487e84ab14082d82b1f803645), right?